src/main/java/com/cloudbees/plugins/credentials/CredentialsStore.java

```diff
@@ -256,7 +261,7 @@ public Domain getDomainByName(@CheckForNull String name) {
      * <li>{@link #updateDomain(Domain, Domain)} </li>
      * </ul>
      *
-     * @return {@code true} iff {@link #addDomain(Domain, List)}
+     * @return {@code true} if {@link #addDomain(Domain, List)}
```

Comment:
“IFF” is a common short-hand for “If and Only If” not a typo ;-)

Reply:
My bad! I was not aware of this. I'll change it back 👍